src/net/http.js

```diff
@@ -56,9 +56,24 @@ Object.assign(pc, function () {
         /**
          * @function
          * @name pc.Http#get
+         * @variation 1
```

Comment:
Have you checked the JSdoc output to see what output from the variation tag looks like? I've never used it before...

Reply:
Of course. Both `d.ts` and `html` output is two separate functions, with and without param `options`.

Reply:
@daredevildave Are any of the variations of the signatures of pc.http functions deprecated/private? This will expose them as public API.

Reply:
seems fine